### PR TITLE
Added the translated version of the Zero trust architecture.

### LINKED ISSUE #1785: [Spanish] localize 'Zero Trust Architecture'
Localize the `content/en/zero-trust-architecture.md` document to Spanish

### PR DESCRIPTION
**Pull request by MartinEliasQ**
### Describe your changes
Added the translated version of the Zero trist architecture.

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1785

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/zero-trust-architecture.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Arquitectura de Confianza cero
+status: Completed
+category: Concepto
+tags: ["security", "", ""]
+---
+
+## ¿Qué es?
+
+La arquitectura de Confianza cero se refiere a un enfoque del diseño y la implementación de sistemas informáticos donde la confianza se elimina por completo.
```
Comment (ramrodo) [inline content/es/zero-trust-architecture.md:10]:
```suggestion
La arquitectura de confianza cero se refiere a un enfoque del diseño y la implementación de sistemas informáticos donde la confianza se elimina por completo.
```

File: content/es/zero-trust-architecture.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+El principio fundamental de 'nunca confiar, siempre verificar' implica que los dispositivos o sistemas, al comunicarse con otros componentes, siempre deben verificarse a sí mismos antes de hacerlo.
+Hoy en día, en muchas redes corporativas, los sistemas y dispositivos internos pueden comunicarse libremente entre sí, ya que se encuentran dentro del límite de confianza del perímetro de dichas redes.
+La arquitectura de Confianza cero toma un enfoque opuesto en el que, a pesar de estar dentro del perímetro de la red, los componentes dentro del sistema tienen que pasar primero por una verificación antes de que se establezca cualquier comunicación.
```
Comment (ramrodo) [inline content/es/zero-trust-architecture.md:13]:
```suggestion
La arquitectura de confianza cero toma un enfoque opuesto en el que, a pesar de estar dentro del perímetro de la red, los componentes dentro del sistema tienen que pasar primero por una verificación antes de que se establezca cualquier comunicación.
```

File: content/es/zero-trust-architecture.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Problema que aborda
+
+En el enfoque tradicional basado en la confianza, donde los sistemas y dispositivos se encuentran dentro de un perímetro de red corporativa, la suposición es que, ya que existe confianza, no hay problemas.
+Sin embargo, la arquitectura de confianza cero reconoce que la confianza es una vulnerabilidad.
+En el supuesto de que un atacante consiga acceso a un dispositivo confiable, el sistema se vuelve vulnerable a un ataque. Esto depende del nivel de confianza y del acceso que se le haya otorgado a dicho dispositivo, pues el atacante, al estar dentro del perímetro 'de confianza' de la red, es capaz de moverse lateralmente a lo largo del sistema.
```
Comment (ramrodo) [inline content/es/zero-trust-architecture.md:19]:
```suggestion
En el supuesto de que un atacante consiga acceso a un dispositivo confiable, el sistema se vuelve vulnerable a un ataque. Esto depende del nivel de confianza y del acceso que se le haya otorgado a dicho dispositivo, pues el atacante, al estar dentro del perímetro "de confianza" de la red, es capaz de moverse lateralmente a lo largo del sistema.
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Algunas sugerencias :)

**Comment by MartinEliasQ** [comment]
> @MartinEliasQ podrías añadir en la descripción del PR (justo debajo de **Related issue number...** la leyenda de `resolves #1785` por favor? Esto para poder relacionar/unir el issue con el PR. Y, una vez fusionado éste PR, se cierre automáticamente el issue ligado.

Listo, @ramrodo

**Review by ramrodo — APPROVED** [review]
LGTM

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**